app/models/viewer.rb

```diff
@@ -11,7 +11,7 @@ class Viewer < ApplicationRecord
             allow_blank: true
 
   def default_viewer_path
-    'oembed_default'
+    'catalog/oembed_default'
```

Comment:
Pretty sure this doesn't break anything. I tested to make sure but I could have missed somewhere.